cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,62 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/login');
   });
 
-  it('', () => {
+  it('user is able to login, add book to collection and delete it', () => {
```

Comment:
You need to have one describe and three it: logging in and navigating to the Book Store, searching for a book and adding it to the collection, and going to the profile page and deleting the book.